### ISSUE TITLE
#define improperly parsed

### ISSUE THREAD

**Issue by Arcnor** [bytedeco/javacpp#184]
For some strange reason, the following `#define` is incorrectly parsed:

```cpp
#define TESTING = "Testing::"
```

...generates...

```java
public static final String TESTING =";
```

If you change the number of colons to be less than 2, it generates correctly. Anything >= 2, it breaks in the described way.

**Comment by saudet** [comment]
I've fixed the issue with the "::" substring in the commit above, but it still won't work in this case because the `=` token is not a string. We need to forget about translating that macro into a variable.

**Comment by saudet** [comment]
Fix released in version 1.3.3. Thanks for reporting!

**Comment by Arcnor** [comment]
Awesome, thanks for fixing!
